swagger_server/controllers/operand_controller.py

```diff
@@ -1,15 +1,29 @@
+import flask
+
+import peewee
+
 from swagger_server import util
+from swagger_server.models.evaluation import Operand
+from swagger_server.util import ValidationError
 
 
-def add_operand():  # noqa: E501
+def add_operand(payload):  # noqa: E501
     """Create an operand
 
     Creates an operand # noqa: E501
 
 
     :rtype: None
     """
-    return 'do some magic!'
+    if not payload:
+        raise ValidationError(400, 'Couldn\'t parse JSON POST body.')
+    if not payload.get('expression_id'):
```

Comment:
Since this guy takes quite a few args in I'd do something like:
```
if not payload.get('expression_id'):
    raise ValidationError(400, 'No expression_id field specified')

if not payload.get('rank'):
    raise ValidationError(400, 'No rank field specified')

if not payload.get('value'):
    raise ValidationError(400, 'No value field specified')

if not payload.get('type'):
    raise ValidationError(400, 'No type field specified')
```